lib/locales/en.ts

```diff
@@ -944,13 +965,17 @@ export const locale: Locale = {
     thorlac_of_iceland_bishop: 'Saint Thorlac, Bishop',
     thursday_of_the_lords_supper: 'Thursday of the Lord’s Supper',
     timothy_of_ephesus_and_titus_of_crete_bishops: 'Saints Timothy and Titus, Bishops',
+    timothy_of_ephesus_and_titus_the_first_bishop_of_crete_bishops:
+      'Saint Titus, the first Bishop of Crete, Saint Timothy, Bishops',
```

Comment:
Same as for `lydia_of_philippi_first_european_christian`.

The locale `timothy_of_ephesus_and_titus_the_first_bishop_of_crete_bishops` is not referenced elsewhere other than in the locale files. As the corresponding definition is already defined in the `proper-of-saints.ts`, you should add a calendar definition in `greece.ts` that extends its definition, and specify the custom locale entry only in this particular calendar:

```ts
// To be added in greece.ts
timothy_of_ephesus_and_titus_of_crete_bishops: {
   customLocaleId: 'timothy_of_ephesus_and_titus_of_crete_bishops',
},
```